### DISCUSSION TITLE
Can I use React-DateTime-Picker as a date picker conditionally?

### DISCUSSION THREAD

**Discussion by TecSachinGupta** [wojtekmaj/react-datetime-picker#182] [Q&A] [answered]
Can we use the DateTime picker as a date picker based on property?
using the condition instead of using 2 different libraries to accomplish the same ?

**Comment by wojtekmaj** [comment, marked as answer]
That is a pretty unusual use case for React-DateTime-Picker and I do recommend using React-Date-Picker, still, but I think you can totally pull this off by providing custom `format` prop, e.g. `"dd.MM.yyyy"`. The only gotcha you need to watch out for is that the hour passed in Date() instance you get in `onChange` will be pretty unpredictable.
